.github/workflows/end-to-end.yml

```diff
@@ -62,6 +64,7 @@ jobs:
         with:
           path: backend/vendor/bundle
           key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+          # Cache not found for input keys: Linux-gems-1249f66ba3e6508ebd38170d96559bbd65919b464e8f37bc4ebc5adb9590e0b8, Linux-gems-
```

Comment:
is this comment useful ?

Reply:
Not anymore since the update of action/cache ! 
Thanks I forget to delete it !